helm/macropower-analytics-panel-server/templates/deployment.yaml

```diff
@@ -33,6 +33,7 @@ spec:
       - name: {{ include "name" . }}
         image: "{{ .Values.registry.domain }}/{{ .Values.server.image.name }}:{{ .Values.server.image.tag }}"
         imagePullPolicy: {{ .Values.server.image.pullPolicy }}
+        args: {{ .Values.server.args }}
```

Comment:
@QuentinBisson is this right? Do I need to do something special here to inject an array?

Reply:
done